packages/fastui/src/components/footer.tsx

```diff
@@ -0,0 +1,42 @@
+import { ClassName, useClassName } from '../hooks/className'
+
+import { LinkProps, LinkComp } from './link'
+import { TextComp, TextProps } from './text'
+
+export interface FooterProps {
+  type: 'Footer'
+  extra_text?: TextProps
+  links: LinkProps[]
+  className?: ClassName
+}
+
+export const FooterComp = (props: FooterProps) => {
+  const links = props.links.map((link) => {
+    link.mode = link.mode || 'footer'
+    return link
+  })
+  const linkProp = useClassName(props, { el: 'link' })
+  const extraSeparatorProp = useClassName(props, { el: 'extra-separator' })
+  const extraProp = useClassName(props, { el: 'extra' })
+  return (
+    <div className={useClassName(props, { el: 'contents' })}>
```

Comment:
Thanks for the review!

I agree and pushed most of the specifics into the `fastui-bootstrap` package.

In addition to what you wrote, I just left the `TextComp` for the `extraText`, as I felt it might be a bit confusing if one could specify it but it wouldn't appear at all in the default component.